docs/installation/application-configuration.md

```diff
@@ -1,71 +1,49 @@
+| `ELASTICSEARCH_HOST`                   | `'elasticsearch:9200'`               | host of your Elasticsearch                                                           |
+| `REDIS_HOST`                           | `'redis'`                            | host of your Redis storage (credentials are not supported right now)                 |
+| `REDIS_PREFIX`                         | `''`                                 | separates more projects that use the same redis service                              |
+| `MAILER_TRANSPORT`                     | `'smtp'`                             | access data of your mail server                                                      |
+| `MAILER_HOST`                          | `'smtp-server'`                      | ...                                                                                  |
+| `MAILER_USER`                          | `null`                               | ...                                                                                  |
+| `MAILER_PASSWORD`                      | `null`                               | ...                                                                                  |
+| `MAILER_DELIVERY_WHITELIST`            | `'/@shopsys\.com$/'`                 | regex text items if you want to have master email but allow sending to specific addresses (set as text separated by comma for multiple values) |
+| `MAILER_DISABLE_DELIVERY`              | `false`                              |set to `true` if you don't want to send any emails                                    |
+| `MAILER_MASTER_EMAIL_ADDRESS`          | `'no-reply@shopsys.com'`             | set if you want to send all emails to one address (useful for development)           |
+| `APP_SECRET`                           | `'ThisTokenIsNotSoSecretChangeIt'`   | randomly generated secret token                                                      |
+| `ELASTIC_SEARCH_INDEX_PREFIX`          | `''`                                 | separates more projects that use the same elasticsearch service                      |
+| `IGNORE_DEFAULT_ADMIN_PASSWORD_CHECK`  | `'0'`                                | set to `true` if you want to allow administrators to log in with default credentials |
+| `OVERWRITE_DOMAIN_URL`                 | `'http://webserver:8080'`            | overwrites URL of all domains for acceptance testing (set to `~` to disable)         |
+| `SELENIUM_SERVER_HOST`                 | `'selenium-server'`                  | with native installation the selenium server is on `localhost`                       |
+| `SHOPSYS_CONTENT_DIR_NAME`             | `'content-test'`                     | web/content-test/ directory is used instead of web/content/ during the tests         |
+| `TRUSTED_PROXIES`                      | `'127.0.0.1'`                        | proxies that are trusted to pass traffic, used mainly for production (set as text separated by comma for multiple values) |
+
```

Comment:
Maybe there would be some info about dumping ENV variables with `php bin/console debug:container --env-vars` command

Reply:
That is right, even it is in symfony docs we could mention it here.

Reply:
Added in 8d21964ae8c0c934939fefb5ff934da24cb351e5